crates/biome_js_semantic/src/semantic_model/reference.rs

```diff
@@ -189,8 +192,16 @@ impl UnresolvedReference {
         &self.data.binding_node_by_start[&reference.range.start()]
     }
 
-    pub fn tree(&self) -> AnyJsIdentifierUsage {
```

Comment:
This will be considered as a breaking change as the binding does not guarantee it to be a `JsIdentifier` anymore. It could be a JSX element or a JSX fragment. Also `biome_js_semantic` releases the crate.